bot/constants.py

```diff
@@ -493,7 +493,7 @@ class Roles(metaclass=YAMLGetter):
 
     contributors: int
     help_cooldown: int
-    muted: int
+    muted: int  # TODO remove when no longer relevant.
```

Comment:
Is this still of relevance because of the potentially existing mutes ?
We could still rename it to "Timed out" to  reflect that someone is timed out in discord, because i think it's quite handy to have that indicated in the user's roles

Reply:
Yeah it's just for the existing mutes. As a moderator it will be easy to see that they're timed out (for example you will be able to remove their timeout in the context menu). Their messages will appear grayed out in chat with a red indicator.

Reply:
Sounds good